### ISSUE TITLE
enet: outdated gnuconfig files

### ISSUE THREAD

**Issue by ax3l** [ornladios/ADIOS2#2587]
According to our conda-forge cross-compilation recipes for aarch64 on macOS (X1), there seem to be outdated `config.sub` and `config.guess` files in `thirdparty/enet/enet/`.

https://github.com/conda-forge/adios2-feedstock/pull/21

**Comment by eisenhauer** [comment]
Yeah, probably, as those were last updated in 2017.  But ADIOS is using the enet CMake configuration to build it.   Are the old configure support files a problem?  If so, easier if we update them or blow away autoconf support?

**Comment by ax3l** [comment]
If we can all rely on CMake logic that would be the easiest, I think.

**Comment by eisenhauer** [comment]
OK.  Let me know if we can close this if that works out.  Otherwise, happy to update or otherwise change.

**Comment by ax3l** [comment]
Sounds good, thanks!
